homeassistant/components/nissan_leaf/binary_sensor.py

```diff
@@ -3,27 +3,26 @@
 
 from homeassistant.components.nissan_leaf import (
     DATA_LEAF, DATA_PLUGGED_IN, LeafEntity)
+from homeassistant.components.binary_sensor import BinarySensorDevice
 
 _LOGGER = logging.getLogger(__name__)
 
 DEPENDENCIES = ['nissan_leaf']
 
 
-def setup_platform(hass, config, add_devices, discovery_info=None):
+def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up of a Nissan Leaf binary sensor."""
-    _LOGGER.debug(
-        "binary_sensor setup_platform, discovery_info=%s", discovery_info)
-
     devices = []
-    for key, value in hass.data[DATA_LEAF].items():
+    for vin, datastore in hass.data[DATA_LEAF].items():
         _LOGGER.debug(
-            "binary_sensor setup_platform, key=%s, value=%s", key, value)
-        devices.append(LeafPluggedInSensor(value))
+            "binary_sensor setup_platform, vin=%s, datastore=%s",
```

Comment:
It's enough to log the vin number. Logging the datastore just prints the instance reference.

Reply:
Fixed logging with commit https://github.com/home-assistant/home-assistant/pull/21145/commits/27d28b0f10cbe70f5f25a3d0b6405dc951b5cc42